packages/netlify/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "../../tsconfig.base.json",
   "include": ["src"],
   "compilerOptions": {
-    "outDir": "./dist"
+    "outDir": "./dist",
+    "types": ["vite/client"]
```

Comment:
IIRC setting types here would make `@types/*` not work, could we use this comment instead on the top of the file that uses `import.meta.env.*`?

```js
/// <reference types="vite/client" />
```